src/components/varsom-avalanche-activity-obs2/varsom-avalanche-activity-obs2.tsx

```diff
@@ -85,13 +86,28 @@ export class VarsomAvalancheActivityObs2 {
     ></varsom-key-value>
     :""}
     
-<br></br>
-<br></br>
+
+<span class="icons-container">
     {this.ValidExposition ? 
-    <div><img src={(`src/assets/svg/ext/${this.ValidExposition}.svg`)}></img></div>
+    <span><img src={(`src/assets/svg/ext/${this.ValidExposition}.svg`)}></img></span>
     :""}  
 
 
+{(this.ExposedHeight1 && !this.ExposedHeight2) ? 
+<span class="avalanche-height-container">
+  <img src={(`src/assets/svg/avalanche/Icon-Avalanche-height.svg`)}></img><span>{this.ExposedHeight1 + "m"}</span></span>
+:""}
+
+{this.ExposedHeight1 && this.ExposedHeight2 ? 
+  <span class="avalanche-height-container">
+  <img src={(`src/assets/svg/avalanche/Icon-Avalanche-height2.svg`)}></img><span class="avalanche-height-number">{this.ExposedHeight1 + "m" + this.ExposedHeight2 + "m"}</span></span>
+:""}
+
+    
+    </span>
+
+  
+
```

Comment:
Denne visningen ser helt lik ut for skredaktivitet og skredfarevurdering. Burde den vært skilt ut i en egen komponent?
OnceAndOnlyOnce vettu;)

Reply:
Ja, bør nok være egen komponent. Lager egen oppgave på det.

Reply:
Her kan du vel ta i bruk den nye komponenten du har laget i #97 ?

Reply:
Har oppdatert med ny komponent